### ISSUE TITLE
Moving files not working after latest update

### ISSUE THREAD

**Issue by rmacari** [AzuraCast/AzuraCast#2062]
**Using Docker installation method**
Yes

**Host Operating System**
Linux Ubuntu

**Describe the bug**

Trying to move files from the root dir to a folder doesn´t work after the latest update.

**To Reproduce**
Go to Music Files
Select a file from the root dir
Click Move
Click on a folder (Should be a Select Button instead)
Click move to Directory
Error Message: An error occurred and your request could not be completed.

**Expected behavior**

Should appear a Select button and file should be moved
**Relevant Logs**
Could not find any error in logs

**Screenshots**
![Screen Shot 2019-10-13 at 21 53 18](https://user-images.githubusercontent.com/5432270/66722338-e96c1d00-ee03-11e9-88ad-14e24dcc47ca.png)

**Device(s):**
 - Device:  
Desktop and iPhone6
 - OS: 
Mac and iOS
 - Browser: 
Firefox and Chrome
 - Version: 
Latest available to OS

**Additional context**

**Comment by SlvrEagle23** [comment]
@rmacari If you click the name of a folder, then click "Move to Directory" when it's listed at the top of the "move" dialog box, that should move the files without any issue. This isn't an error in the design of the modal dialog box; we rebuilt it from the ground up in a new frontend library, so these changes are by design.

**Comment by rmacari** [comment]
It should not. I don´t think you understand the issue. 

First, the logic for moving a file and ANY system is FIRST to select the file, then move the file to the targeted dir, this is the logic, and this is how Azuracast always worked. 

So right now to have the file to the desired folder I have to download the file, click the folder name on the folder list, get inside the folder, then upload the file. It´s a lot of work just to one file in the right location. And please don´t patronize me because before the update I´ve moved files every single day and now I have other users that upload and move files complaining the same issue. 

And although your file moving logic is not logical I´ve tried and obviously didn´t work. 

I´m sending this screenshots to try to show better the issue and any user here that move files should understand the problem. 

![Screen Shot 2019-10-14 at 12 13 04](https://user-images.githubusercontent.com/5432270/66747364-41df0100-ee7c-11e9-8d34-5a3e4ea97989.png)
![Screen Shot 2019-10-14 at 12 13 20](https://user-images.githubusercontent.com/5432270/66747365-41df0100-ee7c-11e9-9cc0-8e8b638b0021.png)
![Screen Shot 2019-10-14 at 12 13 48](https://user-images.githubusercontent.com/5432270/66747366-41df0100-ee7c-11e9-9172-d52ce0838dec.png)
![Screen Shot 2019-10-14 at 12 13 57](https://user-images.githubusercontent.com/5432270/66747367-42779780-ee7c-11e9-84d8-6b948e9712c3.png)
![Screen Shot 2019-10-14 at 12 14 05](https://user-images.githubusercontent.com/5432270/66747369-42779780-ee7c-11e9-94e3-cfd5ac4ad195.png)

**Comment by SlvrEagle23** [comment]
@rmacari It's clear there's a misunderstanding here, but there's no need for a short temper on your part. We are volunteers delivering this project for free to thousands of users around the planet, and doing the best we can, often despite real world issues causing us inconvenience.

I am not saying that the new way of moving files is to download, switch directories, then reupload. That is not required at all. I'm saying we slightly modified the "Move" dialog's design to the new one you're seeing here. We did not remove the "Move" functionality altogether, nor would we.

What you're experiencing is an error with the updated "Move" dialog that is not intentional and that we need to resolve.

In order for us to diagnose the error and resolve it, it would be helpful to open your web inspector (F12 on most browsers), switch to the "Network" tab, then attempt to do the move operation again. You will see a request that returns in red, likely a HTTP 500 error. Let me know what the raw response of that message is and we can fix the issue.

**Comment by rmacari** [comment]
Sorry man, I was just trying to highlight the issue, I was getting frustrated because I know how good this project is and efficiently and helpful you all are in fixing the errors. 

I got this error on that 500 HTTP: 

```
500
Method not allowed. Must be one of: OPTIONS, PUT
RoutingMiddleware.php : 94

home
```

If you need any more info please let me know.

BTW, I know that this work is made by volunteers and if you guys open a Patreon account I will happily send you some money there every, you all deserve. Think about this to fund the project and to help keep this going. Cheers.

**Comment by SlvrEagle23** [comment]
@rmacari Thank you for supplying me with the error message. It turns out one of the last-minute changes we made before shipping the new media manager made the "Move" dialog box stop working properly, but fortunately the solution is an easy one.

I've fixed the problem and published the updated code, which should now be available for you to download.

As for supporting our project, any support would be greatly appreciated! We have links to our Ko-fi and Patreon pages on our [Donate page](https://www.azuracast.com/donate.html).
